### DISCUSSION TITLE
How do I get the Shark achievement?

### DISCUSSION THREAD

**Discussion by Kayke-Fujinaka** [community/community#19899] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't really know what that achievement is about, but does anyone know for sure how to get it?

**Comment by vietdoo** [comment, marked as answer]
Hi~!
Pull Shark check your PR record
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

You can create new branch on your repository, code something.... and merge it to main. 
Do this 2 times and you will get the badge x1

**Reply by Kayke-Fujinaka** [reply]
Hi :)
Thank you very much for the message

**Comment by JaimeTR** [comment]
You have to have open pull requests that have been merged into some repository.
2 pull requests approved and merged and you get the first badge

If I answered your question, please mark as correct answer <3

**Reply by Kayke-Fujinaka** [reply]
Thanks, my friend :)

**Comment by jventura1738** [comment]
Open a pull request, and have it be merged (x2 for the first achievement)

**Comment by Ubaidullahqureshi** [comment]
Galaxy badge : We should  Answer two discussions ! 
Pull Shark badge : We should Have two pull requests merged !
yolo badge : We should Merge code without review !


![173281417-4915e511-7f93-4c15-b7f3-12b9d469411b](https://user-images.githubusercontent.com/103785835/177031447-e6a8da84-fb1c-42d1-b468-0f23a3fbf9e3.png)

**Comment by ascender1729** [comment]
> I don't really know what that achievement is about, but does anyone know for sure how to get it?

You need to open at least two pull requests & merge them.

**Comment by C0ffeeCode** [comment]
I get the achievement for personal private archived repos but not for public ones?

**Comment by MathiasReker** [comment]
Open Pull requests

**Comment by COLLiDER4D** [comment]
Open 2 or more pull requests

**Comment by manasesjesus** [comment]
You need to open pull requests and they have be merged.
So, create some branches in your repos (or contribute to others), open PRs and merged them. You'll get the badge afterwards 🦈.

**Comment by DumbergerL** [comment]
You can also open a pull request on your private repo. That also counts.
